### PR TITLE
Adjust examples to automatic `.wasm` file detection for the `no-modules` target

### PR DESCRIPTION
**Pull request by daxpedda**
Follow-up for #3169.

I couldn't really try it because I can't get wasm-pack to work for me.

### CODE REVIEW

**Comment by daxpedda** [comment]
@Liamolucko friendly ping.

File: examples/wasm-in-web-worker/worker.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const {NumberEval} = wasm_bindgen;
 
 async function init_wasm_in_worker() {
     // Load the wasm file by awaiting the Promise returned by `wasm_bindgen`.
-    await wasm_bindgen('./pkg/wasm_in_web_worker_bg.wasm');
+    await wasm_bindgen();
```
Comment (Liamolucko) [inline examples/wasm-in-web-worker/worker.js:14]: Won't this not work due to #3277? I haven't actually tested this as I'm on my phone right now.

Comment (daxpedda) [reply examples/wasm-in-web-worker/worker.js:14]: Yep ...

Comment (daxpedda) [reply examples/wasm-in-web-worker/worker.js]: Fixed.
Thank you for reviewing all my PRs so attentively.

**Review by Liamolucko — APPROVED** [review]

**Pull request merged**